humanguide/layout/03-main-actions.rst

```diff
@@ -0,0 +1,78 @@
+Main actions
+============
+
+Main actions are displayed in the corresponding header of the Page.
+Trying to be focused on the action relevant to the current page.
+
+Main page of Contacts app shows:
+
+.. figure:: /_static/images/humanguide/27.png
+
+Tapping on a contact, show only relevant actions:
+
+.. figure:: /_static/images/humanguide/28.png
+
+Slots
+-----
+
+The header contains a number of slots that can hold actions or
+navigational options. Depending on the surface or window size,
+additional slots can be added to show the actions otherwise hidden in
+drawers.
+
+.. figure:: /_static/images/humanguide/29.png
+
+Action Drawer
+-------------
+
+An action drawer can be used for when no other slots are available to
+show them. However, when your app is on a larger surface, like on a
+desktop, then actions will appear in the slots.
```

Comment:
"Actions will be placed into an action drawer when there are no available slots to house them..."